website/docs/getting-started/using-podman-mac-helper-to-migrate-from-docker-to-podman-on-macos.md

```diff
@@ -53,7 +53,11 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
      $ podman import <your_container_archive>.tar
      ```
 
-4. Use the `podman-mac-helper` tool to run commands.
+#### Verification
+
+1. Your tools communicating to the Docker socket, such as Maven or test containers, communicate with Podman without reconfiguration.
```

Comment:
maybe add a link to https://www.testcontainers.org/ ? Also not sure about spelling of `test containers` like that

I'm not sure about listing maven maybe say IDE as well as it's more generic as many people are not doing java

Reply:
That's enlightening, I wasn't aware 'test containers' was about that project!

Reply:
Done